storage/postgres/storage_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package postgres
+
+import (
+	"database/sql"
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian/storage/postgres/testdb"
+)
+
+var DB *sql.DB
```

Comment:
This _probably_ doesn't need to be exported, but it'd be worth adding a comment here explaining that it's shared by all tests etc.